receiver/dockerstatsreceiver/metadata.yaml

```diff
@@ -463,6 +469,14 @@ metrics:
       value_type: int
       aggregation_temporality: cumulative
       monotonic: false
+  container.memory.fails:
+    enabled: true
```

Comment:
For new metrics I prefer them not to be enabled by default since this will break downstreams. We already have other memory metrics enabled by default that can indicate memory limits being hit, so I'd probably prefer this to be off-by-default at least initially.

Reply:
This is actually an overlook, I totally agree.